docs/docs-content/vm-management/create-manage-vm/using-datavolumes.md

```diff
@@ -0,0 +1,184 @@
+
+Note 2 important aspects of this DataVolume:
+* It sets an annotation to prevent deleting the `DataVolume` object after if finishes the import process. This is necessary to prevent Palette from continuously recreating the `DataVolume` object, resulting in error messages.
+* The `volumeMode` needs to align with the mode that the CSI supports for `ReadWriteMany` access. For some CSIs such as Rook-Ceph and EMC PowerMax this is `Block` mode, while for other CSIs such as Portworx, Longhorn and EMC PowerFlex this is `Filesystem` mode.
+
+:::info
+When `Filesystem` mode is selected, the CDI will automatically apply a small overhead factor to the PVC size to account for the storage space lost due to filesystem overhead. As a result, the PVC for such a template will be slightly larger. Any clones made from this PVC also need to use this slightly larger minimum size.
+:::
+
+As soon as the `DataVolume` resource above is applied to the cluster, the CDI will begin importing the data into the target PVC in the `vmo-golden-images` namespace. The target PVC will have the same name as the `DataVolume` object.
+
+<br />
+
+### Creating vmTemplates with CSI-assisted cloning
+
+The next step is to create a `vmTemplate` resource that clones the PVC in the `vmo-golden-images` namespace. This is done through the `dataVolumeTemplates` section:
+
+```
+apiVersion: spectrocloud.com/v1
+kind: VmTemplate
+metadata:
+  name: ubuntu-2204
+spec:
+  dataVolumeTemplates:
+    - metadata:
+        name: ubuntu-2204
+      spec:
+        source:
+          pvc: 
+            name: template-ubuntu-2204
+            namespace: vmo-golden-images
+        pvc:
+          accessModes:
+            - ReadWriteMany
+          resources:
+            requests:
+              storage: 50Gi
+          volumeMode: Block      # or "Filesystem", depending on the CSI used
+  template:
+    < rest of template >
+```
+
+The `spec.dataVolumeTemplates.spec` section defines which source PVC to clone and what PVC to clone into. In the example above, the `source` section points to the `template-ubuntu-2204` PVC in the `vmo-golden-images` namespace. The `pvc` section creates a new target PVC with the specified `accessMode` and `volumeMode` characteristics. The size of the new PVC must be the same or larger as the source PVC.
+
+:::info
+To allow CSI-assisted cloning to work, there are a few [prerequisites](https://github.com/kubevirt/containerized-data-importer/blob/main/doc/csi-cloning.md#prerequisites) that must be followed. Most importantly, the source and target PVC must share the same Storage Class and the same Volume Mode. If the requirements are not met, CDI will automatically fallback to the slower host-assisted cloning.
+
+To enable CSI-assisted cloning into larger PVCs than the source PVC, the Storage Class used must have `allowVolumeExpansion` set to `true`.
```

Comment:
Where is this `allowVolumeExpansion` setting? I don't see it in any of your manifests.